restler/restler_settings.py

```diff
@@ -485,6 +485,8 @@ def convert_wildcards_to_regex(str_value):
         self._token_refresh_cmd = SettingsArg('token_refresh_cmd', str, None, user_args)
         ## Interval to periodically refresh the authentication token (seconds)
         self._token_refresh_interval = SettingsArg('token_refresh_interval', int, None, user_args)
+        ## New format for specifying authentication settings 
```

Comment:
>New format for specifying authentication settings  [](http://example.com/codeflow?start=11&length=50)

Suggest 'Set the authentication options'  (we can document in the doc that it's the new format).

Reply:
Added